src/com/palmergames/bukkit/towny/command/TownCommand.java

```diff
@@ -4150,6 +4172,66 @@ public static void parseTownTrustCommand(CommandSender sender, String[] args, @N
 		town.save();
 	}
 
+	public static void parseTownTrustTownCommand(CommandSender sender, String[] args, @Nullable Town town) throws TownyException {
+
+		if (args.length < 1
+			|| args.length < 2 && (args[0].equalsIgnoreCase("add") || args[0].equalsIgnoreCase("remove"))
+			|| args.length == 1 && !args[0].equalsIgnoreCase("list")) {
+			HelpMenu.TOWN_TRUSTTOWN_HELP.send(sender);
+			return;
+		}

```

Comment:
```suggestion
		if (args.length < 1
			|| args.length < 2 && ("add".equalsIgnoreCase(args[0]) || "remove".equalsIgnoreCase(args[0]))
			|| args.length == 1 && !"list".equalsIgnoreCase(args[0])) {
			HelpMenu.TOWN_TRUSTTOWN_HELP.send(sender);
			return;
		}
```

Literal "list", "add" or "remove" are arguments of 'equalsIgnoreCase()', instead of its qualifier

Reply:
Have implemented change as suggested in commit.